src/common/lib/utils.ts

```diff
@@ -45,7 +46,7 @@ const utils = {
   openUrl: (url: string) => {
     browser.tabs.create({ url });
   },
-  openPrompt: <Type>(message: {
+  openPrompt: async <Type>(message: {
```

Comment:
does this need to be async? if this is async I think then line 75 should not `return new Promise`

Reply:
We need to await the async function getWindowPosition, therefore it has to be async.

https://github.com/getAlby/lightning-browser-extension/pull/1883/files#diff-4bc1a33d21997d321e25376ba8b1376f5a25afc2b6fb9450515547c9aec95d66R73

Reply:
```
return getWindowPosition(windowWidth, windowHeight).then(dimensions => {

})
```